benchmark each module in their own node process

hoeck opened the pull request:
trying to fix #864

moltar commented on index.ts:
```diff
@@ -1,4 +1,69 @@
+
+    default:
+      console.error('unknown command:', command);
+      process.exit(1);
```
> Don't use process.exit(); throw an error instead

eslint-ignore it?

moltar commented on test/benchmarks.test.ts:
```diff
@@ -1,5 +1,47 @@
+test('all cases must have been imported in tests', () => {
+  const registeredCases = new Set<string>();
+
+  getRegisteredBenchmarks().forEach(([benchmarkId, benchmarkCases]) => {
```
> 'benchmarkId' is defined but never used

moltar reviewed (changes requested):
Some lint issues but otherwise great work!

tonivj5 commented:
Very excited to see the "real" ones result 👀👏🏻

moltar commented:
Oops, sorry, I've introduced a small conflict by merging #865

hoeck commented:
@moltar fixed all linter issues (atm. I am not using Eslint at work so I always forget about it on other projects :sweat_smile:)

Rebased, added a message linking to this issue and disabled an Eslint rule that was complaining about the way spectypes are imported.

hoeck commented:
Sorry for the noise, I should have used a Github markdown previewer :expressionless: :grinning:.

moltar commented:
Sweet @hoeck! 

> I am not using Eslint at work 

Do you use something else?

sinclairzx81 commented:
@hoeck Good work :) +1

hoeck commented:
> Do you use something else?

No, we're only using prettier in our team of two. There have been always more important tasks than fixing the old and ignored tslint setup :smile: .
